### PR TITLE
feat(manager) : Integrating backstage feedback service | URL Redirect | Spa detailed settings tab hide

### PR DESCRIPTION
**Pull request by nikhitamore**
## Explain the feature/fix

1. This feature integrates user feedback and bug reports from Spaship with a backstage feedback service, enabling automatic creation of Jira issues in the Spaship board when users submit feedback or bugs through Spaship manager. This enhances issue tracking and collaboration for the development team.
2. Allowing user to redirect to specific URL after Login.
3. The button for the SPA detailed settings tab has been hidden to prevent user confusion, as it was mistakenly interpreted as specific settings for SPAs only. This adjustment aims to improve clarity and avoid misleading user expectations regarding the page's functionality.

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

This feature integrates user feedback and bug reports from Spaship with a backstage feedback service, enabling automatic creation of Jira issues in the Spaship board when users submit feedback or bugs through Spaship manager. This enhances issue tracking and collaboration for the development team.

 

Case 1 : Submit Bug from Spaship UI:

image
 
Upon bug submission, an email notification is triggered to notify relevant stakeholders.
image

The bug report is reflected in the backstage feedback service, ensuring it is documented for further processing.Simultaneously, a corresponding Jira issue is automatically created in the Spaship board for detailed tracking and resolution.
image

Case 2 :  Allow user to submit feedback from the screen 

Upon bug submission, an email notification is triggered to notify relevant stakeholders.
image

 mail shall be recieved
image

and same shall be refected in the backstage 
image

 The button for the SPA detailed settings tab has been hidden to prevent user confusion
image

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/src/views/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -23,15 +23,21 @@ import { Nav } from './components/Nav';
 export const LoginPage = (): JSX.Element => {
   const [isLoggingIn, setIsLoggingIn] = useToggle();
   const { status } = useSession();
-  const returnUri = window.location.search;
+  const [returnUri, setReturnUri] = useState('');
+  // const returnUri = window.location.search;
```
Comment (soumyadip007) [inline packages/manager/src/views/LoginPage/LoginPage.tsx:27]: please remove this line

File: packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx [hunk 2]
```diff
@@ -89,11 +90,11 @@ export const SPAPropertyDetailPage = (): JSX.Element => {
                 </TabTitleIcon>
                 <TabTitleText>Settings</TabTitleText>
               </>
-            }
+         
             aria-label="SPA-detailed-Settings"
           >
             {openTab === 3 && <Settings />}
-          </Tab>
+          </Tab> */}
```
Comment (soumyadip007) [inline packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx:97]: Please remove these codes

File: packages/manager/src/config/env.ts [hunk 3]
```diff
@@ -37,5 +41,7 @@ export const env = {
   PUBLIC_SPASHIP_LIGHTHOUSE_URL: process.env.NEXT_PUBLIC_SPASHIP_LIGHTHOUSE_URL || '',
 
   // gitbroker url
-  PUBLIC_SPASHIP_GIT_BROKER_URL: process.env.NEXT_PUBLIC_SPASHIP_GIT_BROKER_URL || ''
+  PUBLIC_SPASHIP_GIT_BROKER_URL: process.env.NEXT_PUBLIC_SPASHIP_GIT_BROKER_URL || '',
+  // ChaturDocs link
```
Comment (soumyadip007) [inline packages/manager/src/config/env.ts:45]: // chaturDocs url

**Review by soumyadip007 — CHANGES_REQUESTED** [review]

File: packages/manager/src/config/env.ts (hunk 3 above)
Comment (nikhitamore) [reply packages/manager/src/config/env.ts]: done

File: packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx (hunk 2 above)
Comment (nikhitamore) [reply packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx:97]: This is a temporary fix, hence had added a NOTE and commented it

File: packages/manager/src/views/LoginPage/LoginPage.tsx (hunk 1 above)
Comment (nikhitamore) [reply packages/manager/src/views/LoginPage/LoginPage.tsx]: done

**Review by soumyadip007 — APPROVED** [review]
LGTM

**Pull request merged**